config/controller.yaml

```diff
@@ -22,6 +22,10 @@ kind: ConfigMap
 metadata:
   name:  lifecycle-image
   namespace: kpack
+  annotations:
+    lifecycle.kpack.io/platformApiVersions: "0.3 0.4"
+  labels:
+    lifecycle.kpack.io/lifecycleImage: "true"
```

Comment:
What is the intent of this label?

Reply:
for the `ValidatingWebhookConfiguration.objectSelector` to scope the webhook to only the one ConfigMap instead of all ConfigMaps

Reply:
Should we start using `app.kubernetes.io/part-of` https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/ similar to tekton?